### ISSUE TITLE
How to get Bounding Boxes?

### ISSUE THREAD

**Issue by tanjary21** [Megvii-BaseDetection/YOLOX#1583]
Hi.

I'm trying to pass an image to YOLOX and get bounding box coordinates of the detected objects in the image frames, together with confidence and class prediction. But the result I'm getting does not seem to make sense.
```from yolox.models.yolox import YOLOX

detector = YOLOX().eval()

with torch.no_grad():
    img = torch.permute(torch.Tensor(mmcv.imread('/DATA/train/000/img1/000001.jpg')), (2,0,1)).unsqueeze(0)
    result = detector(img)

print(result.shape) # gives torch.Size([1, 3549, 85]). How to convert this to detection bboxes?
```
How do I convert the result to bounding box coordinates, confidence scores and class labels? Please help

**Comment by gedance** [comment]
Hello,
For inference,  you can take a look at the "/demo/ONNXRuntime/onnx_inference.py" or the others in the demo folder. :) Cheers!

**Comment by Joker316701882** [comment]
This issue is temporally closed since @gedance has provided the correct answer.
